solidity/test/ConverterUpgrader.js

```diff
@@ -23,7 +23,7 @@ const RESERVE1_BALANCE = new BN(5000);
 const RESERVE2_BALANCE = new BN(8000);
 const TOKEN_TOTAL_SUPPLY = new BN(20000);
 
-const VERSIONS = [9, 10, 11, 23];
+const VERSIONS = [9, 10, 11, 23, 45];
```

Comment:
I might be missing something but it's critical to have a test for 46 => 46 upgrade since that's the only version that tests the new `upgradeReserves` functionality (and I might not remember and we already do that by default...)

Reply:
Oh, we do check the `latest1 version in addition to these explicit versions. I'll rename this to `LEGACU_VERSIONS` to make it clearer

Reply:
See line 208: `describe(`${init.name}(version = ${version || 'latest'}, activate = ${activate}):`, () => {`

Reply:
At least in the past, this test used to run on all versions explicitly specified, along with the latest version (which didn't need to be specified).